website_oca_psc_team/models/project.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Surekha Technologies (https://www.surekhatech.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+from odoo.tools import image_resize_images
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    description = fields.Html()
+
+    github_repository_ids = fields.One2many(
+        comodel_name='project.github.repository',
```

Comment:
Why not a one2may to `github.repository` without the intermediate model?

Reply:
If we do that without intermediate model then, the behaviour will be same like “Contacts & Addresses” tab in Customers. i.e., If we click on “add an item” button then, it goes to create new record in instead of a selection of dropdown and removing record will delete that original record from the database. That's why, I have created an intermediate model and declared github repo as many2one in that model. This way, it will allow selecting github repo as dropdown.

We can go with many2many as well if the record selection order is not imortant.

Reply:
@st-aprajapati What about a one2many field to `github.repository` and `widget="many2many" ` in the view? It would open a search/list view when clicking on _add an item_. It's done in __CRM/Sales team__ members when adding a member. 

![image](https://user-images.githubusercontent.com/16916103/54117000-344a3000-43f0-11e9-9ca9-1b7290dee560.png)

Reply:
Yes, we can go with this.